src/main/scala/ai/privado/exporter/ExporterUtility.scala

```diff
@@ -106,86 +106,7 @@ object ExporterUtility {
     val sizeOfList = _nodes.size
 
     _nodes.zipWithIndex.flatMap { case (node, index) =>
-      val currentNodeModel =
-        convertIndividualPathElement(node, index, sizeOfList, appCache = appCache, ruleCache = ruleCache)
-      if (
-        index == 1 && node.tag.nameExact(Constants.catLevelOne).valueExact(CatLevelOne.DERIVED_SOURCES.name).nonEmpty
-      ) {
-        var typeFullName = Iterator(node).isIdentifier.typeFullName.headOption.getOrElse("")
-
-        // Temporary fix for python to match the typeFullName
-        typeFullName = updateTypeFullNameForPython(typeFullName, isPython)
-
-        // Going 1 level deep for derived sources to add extra nodes
-        taggerCache.typeDeclMemberCache
-          .getOrElse(typeFullName, TrieMap[String, mutable.Set[Member]]())
-          .get(sourceId) match {
-          case Some(members: mutable.HashSet[Member]) =>
-            // Picking up only the head as any path to base is sufficient
-            val member: Member     = members.head
-            var typeFullNameLevel2 = member.typeFullName // java.lang.string
-            // Temporary fix for python to match the typeFullName
-            typeFullNameLevel2 = updateTypeFullNameForPython(typeFullNameLevel2, isPython)
-
-            taggerCache.typeDeclMemberCache
-              .getOrElse(typeFullNameLevel2, TrieMap[String, mutable.Set[Member]]())
-              .get(sourceId) match {
-              case Some(member2Set: mutable.HashSet[Member]) =>
-                // Picking up only the head as any path to base is sufficient
-                val member2 = member2Set.head
-                // Going 2 level deep for derived sources to add extra nodes
-                convertIndividualPathElement(
-                  member2,
-                  messageInExcerpt = generateDSMemberMsg(member2.name, typeFullNameLevel2),
```

Comment:
1 Thing which we missed. Currently we add the message of member of XYZ class  for original sources. Yours changes have overridden them. Add them to your PR

Use the `generateDSMemberMsg`